generated/nidcpower/nidcpower/session.py

```diff
@@ -4822,11 +4822,28 @@ def fetch_multiple(self, count, timeout=hightime.timedelta(seconds=1.0)):
 
         '''
         import collections
-        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
-
-        voltage_measurements, current_measurements, in_compliance = self._fetch_multiple(timeout, count)
-
-        return [Measurement(voltage=voltage_measurements[i], current=current_measurements[i], in_compliance=in_compliance[i]) for i in range(count)]
+        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance', 'channel'])
+
+        voltage_measurements, current_measurements, in_compliances = self._fetch_multiple(timeout, count)
+
+        with _NoChannel(session=self):
+            all_channels_in_session = self._get_channel_names(range(self.channel_count))
+
+        channel_names = _converters.expand_channel_string(
+            self._repeated_capability,
+            all_channels_in_session
+        )
+        assert len(channel_names) == 1
```

Comment:
`assert len(channel_names) == 1, "fetch_multiple only supports one channel at a time"`

Reply:
I have added the assertion error message in my latest commit.